### ISSUE TITLE
Allow linking to Schedule 70 contract

### ISSUE THREAD

**Issue by ajb** [presidential-innovation-fellows/apps-gov#101]
Every S70 contract has a URI associated with it, so it seems like a good candidate for an object instead of an array, e.g.

```
{ "s70":  "http://www.gsaelibrary.gsa.gov/ElibMain/contractorInfo.do?contractNumber=GS-35F-142DA&contractorName=DEPARTMENT+OF+BETTER+TECHNOLOGY%2C+INC.+THE&executeQuery=YES" }
```

**Comment by stroupaloop** [comment]
good call and this should actually be a requirement instead of an option, will make the update. I didn't see a direct way to search for vendors on the schedule through the website, am I missing something? This way I can find the existing ones in the current listing of companies reporting they are on Schedule 70.

**Comment by stroupaloop** [comment]
another issue that may take some time with this change is the conversion from 

```
contracts: [
    "micro",
    "s70"
]
```

to

```
contracts: {
    "micro": true,
    "s70": https://url.com
}
```

when being applied to the current list of products

**Comment by ajb** [comment]
> I didn't see a direct way to search for vendors on the schedule through the website, am I missing something?

Yeah, search & permalinks aren't something that gov't IT does very well... I bet someone else will know, though.

> another issue that may take some time with this change is the conversion...

Seems like there's a case for a script that can loop through each product listing, read the JSON, make arbitrary changes, and write out the new JSON. The only downside to modifying the JSON structure would be having to fix any PRs that are in-progress, but it wouldn't be too bad...
